project-base/config/packages/snc_redis.yaml

```diff
@@ -23,6 +23,13 @@ snc_redis:
             type: 'phpredis'
             alias: 'session'
             dsn: 'redis://%redis_host%'
+        friendly_url_slugs:
+            type: 'phpredis'
+            alias: 'friendly_url_slugs'
+            dsn: 'redis://%redis_host%'
+            logging: true
```

Comment:
can you explain why logging is here?

Reply:
Yeah, I forgot this one from testing. Now it is removed.